canonical-websites/www.ubuntu.com
# Move contribute to thank you

## Done
Move the contribution section to the download thank you page and removed the logical with the pre-download step.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Follow the download flow to desktop
- See that you don't get the contribute page before the thank you page
- See that the thank you page contains the contribute section
- Try out the contribution
- Return to /download/desktop/thank-you (which is the return URL) and see that the messaging is different and a download does not start.
